### ISSUE TITLE
Add a web editor for any YAML / JSON file

### ISSUE THREAD

**Issue by LeehamElectronics** [kangarko/Synchronize#2]
Hey Matej,
This is a great plugin I will try it out some time today. Iv been thinking about this idea ever since you first mentioned the plugin.
My thinking is, there are a few other plugins out there that achieve the same thing as this plugin, however what would REALLY set this plugin apart is if there were a command like this:

/sync editor [auto tab completing server file directory] [optional -p <server-group>

And the "-p <server-group>" could be like a "force push" to a custom group of defined servers.

Upon typing this command a link is generated and the user can modify the file in there web browser, click save, and the files are automatically saved and synced in the server group.
This is similar to how LuckyPerms lets u modify permissions in a web editor.

It's just an idea, let me know what you think.
Thanks

**Comment by kangarko** [comment]
Thank you!

**Comment by LeehamElectronics** [comment]
> Thank you!

No worries, what do you think of it? Is that something feasible?

**Comment by kangarko** [comment]
Yes of course, however at this stage of the development and with the other things on my plate, all I can say is that I am definitely letting this opened.

I'll rebuild the team we had this year with new talents that will create more time and be able to take on these features. I think it's going to be an awesome one!
